tars/transport/tcphandler.go

```diff
@@ -21,6 +23,8 @@ type tcpHandler struct {
 	tcpNoDelay  bool
 	jobs        chan *connectionHandler
```

Comment:
Should be remove ,also in   https://github.com/TarsCloud/TarsGo/blob/master/tars/transport/tarsserver.go#L62 about jobs stuff.